ci-operator/config/stackrox/stackrox/summarize.py

```diff
@@ -0,0 +1,590 @@
+#!/usr/bin/env -S bash -c 'python3 -m venv "$(dirname "$0")"/.venv && source "$(dirname "$0")"/.venv/bin/activate && pip3 --require-virtualenv --no-input --disable-pip-version-check install PyYAML==6.0 >/dev/null && exec python3 "$0"'
```

Comment:
Looks like this may be confusing GitHub's syntax highlighting but I don't want to give it up unless there are some serious bugs in it (e.g. does not function on MacOS and no way to fix it).